webapp/components/PostCard/PostCard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <ds-card
+    :lang="language"
```

Comment:
```suggestion
    :lang="post.language"
```

Reply:
I can follow your argumentation, and I would prefer your solution if there were not those 18k posts without language. This would lead to something like `lang="null"` and I am not sure is this is what we want. I think it is up to the team to decide which is the better way to handle this problem.